src/plotter_disk.hpp

```diff
@@ -36,8 +36,18 @@
 #define _HAS_STD_BYTE 0
 #define NOMINMAX
 
+#ifdef __APPLE__
+// std::filesystem is not supported on Mojave
 #include "../lib/include/filesystem.hpp"
 namespace fs = ghc::filesystem;
+#else
+#if __has_include(<filesystem>)
+#include <filesystem>
+#else
+#include <experimental/filesystem>
+#endif
```

Comment:
you still need to alias the namespace, don't you?

For example:

```
namespace fs = std::experimental::filesystem;
```

Reply:
seems like filesystem already defines it